R/later.R

```diff
@@ -47,28 +51,52 @@
 #'
 #' @param loop A handle to an event loop.
 #' @param expr An expression to evaluate.
-#' @param autorun Should this event loop automatically be run when its parent
-#'   loop runs? Currently, only FALSE is allowed, but in the future TRUE will
-#'   be implemented and the default. Because in the future the default will
-#'   change, for now any code that calls \code{create_loop} must explicitly
-#'   pass in \code{autorun=FALSE}.
+#' @param autorun This exists only for backward compatibility. If set to
+#'   \code{FALSE}, it is equivalent to using \code{parent=NULL}.
+#' @param parent The parent event loop for the one being created. Whenever the
+#'   parent loop runs, this loop will also automatically run, without having to
+#'   manually call \code{\link{run_now}()} on this loop. If \code{NULL}, then
+#'   this loop will not have a parent event loop that automatically runs it; the
+#'   only way to run this loop will be by calling \code{\link{run_now}()} on this
+#'   loop.
 #' @rdname create_loop
 #'
 #' @export
-create_loop <- function(autorun = NULL) {
-  if (!identical(autorun, FALSE)) {
-    stop("autorun must be set to FALSE (until TRUE is implemented).")
-  }
-
+create_loop <- function(parent = current_loop(), autorun = NULL) {
   id <- .globals$next_id
   .globals$next_id <- id + 1L
-  createCallbackRegistry(id)
+
+  if (!is.null(autorun)) {
+    # This is for backward compatibility, if `create_loop(autorun=FALSE)` is called.
+    parent <- NULL
+    warning("create_loop(autorun=FALSE) is deprecated. Please use create_loop(parent=NULL) from now on.")
```

Comment:
Let's not warn here--if people have CRAN packages calling `create_loop(autorun=FALSE)` let's not make them resubmit just to squelch the warning